databases/redis-enterprise-operator/README.md

```diff
@@ -0,0 +1,7 @@
+# Deploying a Redis cluster with Redis Enterprise operator on GKE
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=cloudshell/tutorial.md&cloudshell_workspace=gke-redis-enterprise-operator)
+
+This example shows how to build and deploy a PostgreSQL cluster using [Kubernetes](https://kubernetes.io) and Day 1 & 2 operations such as Monitoring with [Cloud Monitor](https://cloud.google.com/monitoring) and disaster recovery with [Backup for GKE](https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/concepts/backup-for-gke).
```

Comment:
This doesn't seem accurate to this sample

Reply:
My bad, fixed. Still, URL needs @melissakendall approval because currently, it's fictitious.

Reply:
Which URL? Both Cloud Monitor (should probably be Cloud Monitoring) and Backup for GKE links are correct here

If you are asking what the URL will be for the Redis Enterprise tutorial, the current proposed URL is: https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workloads/enterprise-redis